client/state/inline-help/actions.js

```diff
@@ -22,6 +23,21 @@ import {
 import getContextualHelpResults from 'state/inline-help/selectors/get-contextual-help-results';
 import 'state/inline-help/init';
 
+/**
+ * Set the query search in the sate tree for the inline help.
+ *
+ * @param {string} searchQuery - query store to store.
+ * @returns {Function}            Action thunk.
+ */
+export function setInlineHelpQuerySearch( searchQuery = '' ) {
```

Comment:
Again we'd need to rename this to 

```suggestion
export function setInlineHelpSearchQuery( searchQuery = '' ) {
```